examples/List.links.tsx

```diff
@@ -3,25 +3,25 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 import * as React from 'react';
-import { List, ListItem, LinkAction } from '@itwin/itwinui-react';
+import { List, ListItem } from '@itwin/itwinui-react';
 
 export default () => {
   return (
     <List>
       <ListItem actionable>
```

Comment:
Should all these `ListItem`s have `actionable`? Since the hover styling also come from the `LinkAction`:

https://github.com/iTwin/iTwinUI/blob/b0d4ea174b2d644ee85147ff52d61555804e0f6f/packages/itwinui-css/src/menu/mixins.scss#L52-L53

Reply:
I think lets leave it, because it can't hurt, will also work for older browsers, and will help in case one of the list items is supposed to be actionable but doesn't include a `ListItem.Action`.